docs/static/governance/avm-standard-github-labels.csv

```diff
@@ -6,7 +6,7 @@ Needs: Author Feedback :ear:,Awaiting feedback from the issue/PR author,F18A07
 Needs: External Changes :hammer_and_pick:,When an issue/PR requires changes that are outside of the control of the module. e.g. to an RP.,DE389D
 Needs: More Evidence :balance_scale:,We are looking for more evidence to make a decision on this,F64872
 Needs: Triage :mag:,Maintainers need to triage still,FBCA04
-Needs: Module Owner :mega:,This module needs an owner to develop or maintain it,FF0019
+Needs: Module Owner :mega:,"This module needs an owner to develop or maintain it, or the module owner needs to approve a PR",FF0019
```

Comment:
```suggestion
Needs: Module Owner :mega:,This module needs an owner to develop or maintain it, or the module owner needs to approve a PR,FF0019
```

Reply:
Removing the double quotes as none of the other entries have it and I'm not sure if this would impact the automatic parsing script. Otherwise, it looks good.

Reply:
@ReneHezser what about 
```suggestion
Needs: Module Owner :mega:,This module needs an owner to develop or maintain it, or the module owner needs to review a PR,FF0019
```

Just would like to avoid the wrong impression that the PR has been reviewed and just needs approval

Reply:
I changed the wording, but left the `"`s as we need them for the `,` in the description.

Reply:
Right, good point; I overlooked that. :)

Reply:
Works for me. Will leave @matebarabas to resolve, since he originally opened this comment